components/ui/TextField.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "@emotion/styled";
+
+
+const TextField = styled.input`
+  width: 30px;
+  border-radius: 25px;
+  font-size: 16px;
```

Comment:
`font-size` is one of a few CSS properties that should use `rem` unit. `1rem` in this case.